src/handlebars/faq.handlebars

```diff
@@ -0,0 +1,187 @@
+    <DD> AdoptOpenJDK ships binaries based on the PSU (Patch Set Update) release for the quarterly updates. This
+         contains additional fixes from the CPU (Critical Patch Update) and due to the very high levels of testing
+         which the AdoptOpenJDK projects runs on our release builds we are confident enough that the additional
+         fixes in the PSU updates will not cause any problems.
+    <DT> <strong>How can I get commercial support for AdoptOpenJDK binaries</strong>
+    <DD> The AdoptOpenJDK project does not itself provide commercial support for the binaries we produce,
+         however if you find and issue please raise it in the openjdk-support respository. If you need a stronger
+         level of formal support, check out <a href="https://adoptopenjdk.net/support.html">the support pages</a> for the current list
+         of commercial support providers for the AdoptOpenJDK binaries
+    <DT> <strong>Do you provide 32-bit builds?</strong>
+    <DD> Most of our builds are 64-bit only. The exceptions are arm32 so you can run our builds on your raspberry
+         pi and other such devices, and HotSpot windows builds are aos available in 32-bit versions. A Windows
+         32-bit version is also available for OpenJ9 but only for JDK8. We have had requests for
+         <a href="https://github.com/AdoptOpenJDK/openjdk-build/issues/986">32-bit Linux/x86</a> but they are not
+         available in our build farm at the moment.
+    <DT> <strong>I want to contribute to the project but I'm lost - where do I start?</strong>
+    <DD> Glad you asked - please do! An initial overview can be found in the readme in the
+         <a href="https://github.com/AdoptOpenJDK/TSC/">TSC repository</a>. Also we suggest joining the
+         <a href="https://adoptopenjdk.net/slack.html">Slack channels</a> as a lot of the discussion happens in
+         those as well as github issues
+    <DT> <strong>So what is the TSC?</strong>
+    <DD> They are our overlords :-) Seriously, the TSC (Technical Steering Committee) is a group of people who
+         run the AdoptOpenJDK project. Despite the name, most technical decisions are taken outside the TSC and
+         only escalated to the TSC if issues arise. You will find that they are frequently the people responding
+         to issues in github and also in slack. Business decisions, the scope of the project, and management of
+         finances is all done by the TSC. If you want something discussed by the TSC or need a decision from them
+         please raise an issue in the <a href="https://github.com/AdoptOpenJDK/TSC</a>github TSC</a>repository (or
+         add the <tt>TSC-agenda</tt> label elsewhere) as that will generally flag it for inclusion in a future
+         TSC meeting. The current TSC members with their affiiliation are listed
+         <a href="https://github.com/AdoptOpenJDK/TSC/#tsc-members">here</a>. The TSC members also approve all
+         releases.
+    </DL>
+    </div>
```

Comment:
Another question that comes to my mind: "Where can I get the sources of a specific build / version?"

Reply:
It's a great question and certainly an FAQ ... At the moment we don't have a simple answer though (especially in the general case for both HotSpot and OpenJ9) ... The "source code" link in [any given release](https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/tag/jdk8u212-b03) tempts you into thinking it might be the source, but isn't.

But yeah, something about the `-ga` tags and the `openj9-xxx` branches would be useful to have in